Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/magnum.yml

```diff
@@ -20,6 +20,28 @@
       types:
         Piercing: 24
 
+- type: entity
+  id: BulletMagnumHC
+  name: bullet (.45 magnum)
```

Comment:
Your naming this bullet while the other ones use cartridge.

Reply:
Not really sure what you mean here.
~~tbf you're probably pointing out whats causing the linter failures~~